update cmr launchpad and cumulus api launchpad in terraform (#1301)
nasa/cumulus: docs/additional-deployment-options/cumulus_api_launchpad_authentication.md

1. Upload PKI certificate to S3:

    - Upload the PKI certificate `.pfx` file to S3
    - Use `system_bucket` as bucket name and `<prefix>/crypto/launchpad.pfx` as key

>       - `system_bucket` and `<prefix>` are configured in `cumulus-tf/terraform.tfvars`.

Reviewer: ```suggestion
      - `system_bucket` and `prefix` are configured in `terraform.tfvars` for your [Cumulus deployment](../deployment/README.md#configure-and-deploy-the-cumulus-tf-root-module)
```

>       - If a different private key file name other than `launchpad.pfx` is used, specify it in the `launchpad_certificate` configuration in `cumulus-tf/terraform.tfvars`.

Reviewer: ```suggestion
      - If a different private key file name other than `launchpad.pfx` is used, specify it in the `launchpad_certificate` configuration in `terraform.tfvars` for your Cumulus deployment.
```

> 2. Provide passphrase of PKI certificate in `cumulus-tf/terraform.tfvars`

Reviewer: ```suggestion
2. Provide passphrase of PKI certificate in `terraform.tfvars` for your [Cumulus deployment](../deployment/README.md#configure-and-deploy-the-cumulus-tf-root-module)
```

    ```text
      launchpad_passphrase=<LAUNCHPAD_PASSPHRASE>
    ```

> 3. Configure the `oauth_provider` and `oauth_user_group` configuration parameters in `cumulus-tf/terraform.tfvars`.

Reviewer: ```suggestion
3. Configure the `oauth_provider` and `oauth_user_group` configuration parameters in `terraform.tfvars` for your [Cumulus deployment](../deployment/README.md#configure-and-deploy-the-cumulus-tf-root-module).
```

>    - `oauth_provider` in `cumulus-tf/terraform.tfvars` **must be set to `launchpad` to use Launchpad authentication**.

Reviewer: ```suggestion
   - `oauth_provider` in `terraform.tfvars` **must be set to `launchpad` to use Launchpad authentication**.
```

>    - _Example configuration of the `oauth` variables can be found in Cumulus core's [example](https://github.com/nasa/cumulus/blob/master/example/deployment/sandbox.tfvars)_

Reviewer: ```suggestion
   - _Example configuration of the `oauth_user_group` variable can be found in Cumulus core's [example](https://github.com/nasa/cumulus/blob/master/example/deployment/sandbox.tfvars)_
```

    - Upload the PKI certificate `.pfx` file to S3
    - Use `{{system_bucket}}` as bucket name and `{{prefix}}/crypto/launchpad.pfx` as key
      - `{{system_bucket}}` and `{{prefix}}` are configured in `cumulus-tf/terraform.tfvars`.

>       - If a different private key file name other than `launchpad.pfx` is used, specify it in the `launchpad` configuration in `cumulus-tf/terraform.tfvars`.

Reviewer: Again, the variable to update is `launchpad_certificate`, correct?
